Levels/NewStarbase/NewStarbase.prefab

```diff
@@ -5538,7 +5538,8 @@
                     "Id": 110022581788315572,
                     "m_template": {
                         "$type": "MultiplayerSample::NetworkMatchComponent",
-                        "RespawnPenaltyPercent": 20
+                        "RespawnPenaltyPercent": 20,
```

Comment:
Make sure this merges with Darrin's level revisions.. I think he had a verbal lock on this file.

Reply:
I checked with him, he said i was free to add this